tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_component.ng.html

```diff
@@ -15,10 +15,23 @@
 limitations under the License.
 -->
 
-<div>
-  <div>Debugging</div>
-  <div>
-    <span>Alerts</span>
+<div class="alerts-container">
+  <div class="debugging-title">Debugging</div>
+  <div class="num-alerts-container">
+    <div class="num-alerts-label">Alerts</div>
+    <div class="num-alerts-value" [ngClass]="[numAlerts > 0 ? 'non-zero' : '']">
+      {{ numAlerts }}
+    </div>
+  </div>
+  <div class="alerts-breakdown-container">
+    <div *ngFor="let breakdown of alertsBreakdown;">
```

Comment:
in the future: you may want to consider using semantic tags for accessibility such as ul and li (use CSS to style list away)